backend/src/agents/report_agent.py

```diff
@@ -9,25 +10,26 @@
 
 
 class ReportAgent(Agent):
-    async def create_report(self, file_content: str, materiality_topics: dict[str, str]) -> str:
+    async def create_report(self, file: LLMFile, materiality_topics: dict[str, str]) -> str:
         user_prompt = engine.load_prompt(
             "create-report-user-prompt",
-            document_text=file_content,
             materiality_topics=materiality_topics
         )
 
         system_prompt = engine.load_prompt("create-report-system-prompt")
 
-        return await self.llm.chat(self.model, system_prompt=system_prompt, user_prompt=user_prompt)
+        return await self.llm.chat_with_file(
+            self.model,
+            system_prompt=system_prompt,
+            user_prompt=user_prompt,
```

Comment:
Recently I have started just initialising `user_prompt` and `ssytem_prompt` directly as an arg in the `chat` calls. I don't know why we split these out as separate variables, don't think it adds much.